permissions/plugin-teamscale-upload.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "teamscale-upload"
+github: "jenkinsci/teamscale-upload-plugin"
+paths:
+- "org/jenkins-ci/plugins/teamscale-upload"
```

Comment:
Your path is actually incorrect, it should be the `groupId` item (https://github.com/jenkinsci/teamscale-upload-plugin/blob/master/pom.xml#L10) with periods replaced with forward slashes and then your `artifactId`, so in this case, you should have `"io/jenkins/plugins/teamscale-upload"` as the path.